main.go

```diff
@@ -84,3 +89,28 @@ To reduce security risk, use an unprivileged user account instead.`))
 		os.Exit(exitCode)
 	}
 }
+
+func checkLocaleSettings(ctx context.Context) {
+	// run the locale command
+	output, err := exec.Command("locale").CombinedOutput()
+	if err != nil {
+		fmt.Printf("Error while checking locale settings %v", err.Error())
+		return
+	}
+	// store the value of LC_CTYPE
+	lc_val, err := exec.Command("bash", "-c", "locale | grep LC_CTYPE").Output()
+	if err != nil {
+		fmt.Printf("Error while checking locale settings %v", err.Error())
+		return
+	}
+
+	// check for cannot set LC_CTYPE error
+	flag := strings.Contains(string(output), "Cannot set LC_CTYPE to default locale")
+
+	// if there is a cannot set LC_CTYPE error, exit with an error message
+	if flag {
+		utils.ShowError(ctx, fmt.Errorf(`Failed to initialize database as the default langpack(%s) is not installed. 
+To fix, either set environment variable LC_ALL to 'C' or 'POSIX' or install the langpack %s. [https://www.postgresql.org/docs/current/multibyte.html]`, strings.TrimSpace(string(lc_val)), strings.TrimSpace(strings.Trim(string(lc_val), "LC_CTYPE="))))
```

Comment:
pull out `strings.TrimSpace(string(lc_val)` and `strings.TrimSpace(strings.Trim(string(lc_val), "LC_CTYPE=")))` into variables

Reply:
also there is inconsistent use of quotes in the output - single for `'C` and `'POSIX'` and double for `"en_US.UTF-8"`